config/rhoai/manager_metrics_patch.yaml

```diff
@@ -9,6 +9,9 @@ spec:
       containers:
       - name: manager
         ports:
-        - containerPort: 8443
+        - containerPort: 8080
```

Comment:
in theory, you can remove the entry for 8080 entirely.
but it is fine to leave it there